core/arch/arm/mm/core_mmu.c

```diff
@@ -52,7 +52,7 @@
 
 #include "core_mmu_private.h"
 
-#define RES_VASPACE_SIZE	(CORE_MMU_PGDIR_SIZE * 10)
+#define RES_VASPACE_SIZE	(CORE_MMU_PGDIR_SIZE * CFG_MMAP_RES_REGION_ENRTY_NUMS)
```

Comment:
This is not related to the problem fixed below, please split into two patches.

Reply:
If va space shall be configurable, rather introduce a `CFG_RESERVED_VASPACE_SIZE`.

Reply:
new patches uploaded. thanks for reviewing.